models/config.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package clustermodel
+package models
 
 type Configuration struct {
 	BackupWorkerEnabled            int              `json:"backup_worker_enabled"`
```

Comment:
on line 27 is an error in this file it has a double `""` at the end of the line

Reply:
Great catch, thanks!